src/transforms/util/runtime_transform.rs

```diff
@@ -81,6 +81,7 @@ where
         Box::pin(
             input_rx
                 .map(Message::Process)
+                .fuse()
```

Comment:
Fixes a bug where if no events are sent before shutting down this, then `into_future` will poll this and get `None` which it will pass with the rest of the stream which will be polled again later therefor breaking the stream contract of not calling it after first `None`.